code/game/objects/items/weapons/implants/implant.dm

```diff
@@ -200,6 +202,19 @@
 
 	return 1
 
+/**
+ * Check that we can actually implant this before implanting it
+ * * source - The person being implanted
+ * * user - The person doing the implanting
+ *
+ * Returns
+ * TRUE - Don't implant!
+ * FALSE - I could care less, implant it, maybe don't. I don't care.
+ */
+/obj/item/implant/proc/cancel_implant(mob/source, mob/user)
+	return FALSE
```

Comment:
change this to `can_implant()` cancel implant seems like it's performing an action instead of checking to see if you can implant someone or not